src/main/kotlin/com/terraformation/backend/accelerator/db/ProjectAcceleratorDetailsStore.kt

```diff
@@ -41,13 +41,19 @@ class ProjectAcceleratorDetailsStore(
             landUseModelTypesMultiset,
             PROJECT_ACCELERATOR_DETAILS.asterisk(),
             PROJECTS.COUNTRY_CODE,
+            PROJECTS.CREATED_BY,
+            PROJECTS.CREATED_TIME,
             PROJECTS.ID,
+            PROJECTS.MODIFIED_BY,
+            PROJECTS.MODIFIED_TIME,
+            PROJECTS.NAME,
         )
         .from(PROJECTS)
         .leftJoin(PROJECT_ACCELERATOR_DETAILS)
         .on(PROJECTS.ID.eq(PROJECT_ACCELERATOR_DETAILS.PROJECT_ID))
         .leftJoin(COUNTRIES)
         .on(PROJECTS.COUNTRY_CODE.eq(COUNTRIES.CODE))
+        .where(PROJECTS.ID.eq(projectId))
```

Comment:
The thing that bugs me most about this getting through is that it wasn't caught in a unit test.

Reply:
I was able to reproduce by adding another project in the test, perhaps in the future we should always ensure there are other records in "fetch one" tests.